scripts/performance/common.py

```diff
@@ -80,6 +80,12 @@ def helixpayload():
     '''
     return environ.get('HELIX_CORRELATION_PAYLOAD')
 
+def helixuploadroot():
```

Comment:
This function is not use currently. It was created for copy back job contain from helix machine as more diagnostic info. Like to hear feedback if keeping it around maybe helpful.

Reply:
It's fine to leave it. Artifacts like this are good pointers/reminders that we have such things.